comment/exceptions.py

```diff
@@ -0,0 +1,14 @@
+from rest_framework.exceptions import APIException
+from rest_framework.status import HTTP_400_BAD_REQUEST
+
+
+class CommentBadRequest(APIException):
+    status_code = HTTP_400_BAD_REQUEST
+    default_detail = 'Bad Request'
+
+    def __init__(self, detail=None, status_code=None):
+        if status_code:
+            self.status_code = status_code
+        if not detail:
+            detail = self.default_detail
+        self.detail = detail
```

Comment:
I think you probably forgot to return the translated string here.

Reply:
I didn't. the translated string is already used in json response.

Reply:
Could you please point out the line in the code? I didn't seem to find it right now.

Reply:
could point to the line in the code that does it? as of now, i can't seem to find it.

Reply:
https://github.com/Radi85/Comment/pull/67/files/d40fdf2aa1a3a7996ebaea1600d7824d90daf0e6#diff-75aac02b8fa19905987d843be8ad2212R26

Reply:
correct me if i am wrong, but this wouldn't work for the API?

Reply:
You are right. But I don't think the api should be translated since it will be handled by the developer in the frontend not the end user.

Reply:
yes you're right the frontend developers can translate it on the frontend, but it would be more wise to send it from the backend. They can obviously not use translation if they don't want.

Also, if we go by this principle, we already send translated string responses at the various places in the API.

Reply:
I think we should open a new issue to integrate translations and with this we can cover everything. And we might start adding the translation files for some languages.

Reply:
yes, i think most of the backend response is translated if i can recall correctly. only strings inside the template and js files need to be translated.